src/app/drive/components/FileViewer/FileViewerWrapper.tsx

```diff
@@ -84,34 +84,35 @@ const FileViewerWrapper = ({
   const onDownload = () => currentFile && dispatch(storageThunks.downloadItemsThunk([currentFile as DriveItemData]));
 
   useEffect(() => {
-    setBlob(null);
-    dispatch(uiActions.setFileViewerItem(currentFile));
-    if (currentFile && !updateProgress && !isDownloadStarted) {
-      setIsDownloadStarted(true);
-      fileContentManager
-        .download()
-        .then((downloadedFile) => {
-          setBlob(downloadedFile.blob);
-          setUpdateProgress(0);
-          setIsDownloadStarted(false);
-          if (downloadedFile.shouldHandleFileThumbnail) {
-            handleFileThumbnail(currentFile, downloadedFile.blob).catch(errorService.reportError);
-          }
-        })
-        .catch((error) => {
-          if (error.name === 'AbortError') {
-            return;
-          }
-          console.error(error);
-          setBlob(null);
-          errorService.reportError(error);
-          setIsDownloadStarted(false);
-        });
+    if (currentFile?.plainName === currentFile.name) {
```

Comment:
Why this condition?

Reply:
Because when user changes the name, this is different and is the criteria I found for not reloading the file if the user only changed the name.